### DISCUSSION TITLE
Welcome to myems Discussions!

### DISCUSSION THREAD

**Discussion by guangyuzhang** [MyEMS/myems#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wins2020** [comment]
How can i use your EMS??

actually, our company start a business about EMS and we are looking for code source 
if you can help us develop one we can have a great business

**Comment by guangyuzhang** [comment]
Hi wins2020,
Thanks for your interest in MyEMS.
MyEMS is an industry-leading open source Energy Management System. 
You can start from the [README](https://github.com/MyEMS/MyEMS/blob/master/README_EN.md#myems-components-community-edition) to learn how to download the source code and install it yourself.

https://github.com/MyEMS/MyEMS/blob/master/README_EN.md#myems-components-community-edition

**Comment by wins2020** [comment]
It's necessary to have Linux system or just windows it's working?

**Reply by guangyuzhang** [reply]
Ubuntu Server

**Comment by wins2020** [comment]
Thanks for your email
